.changeset/stale-poems-brush.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-catalog-react': patch
+'@backstage/plugin-scaffolder': patch
```

Comment:
The two packages have reached 1.0 so new additions should be `minor` bumps.

This should probably be split up into two changesets as parts of the changelog entry will be very scaffolder specific.